extensions/amp-bind/0.1/amp-state.js

```diff
@@ -43,7 +43,8 @@ export class AmpState extends AMP.BaseElement {
   activate(invocation) {
     const event = invocation.event;
     if (event && event.detail) {
-      this.updateState_(event.detail.response, /* isInit */ false);
+      this.updateState_(event.detail.response,
```

Comment:
Yea, I was contemplating the same but a bit hesitant to remove functionality that's been in prod. Another reason to do this is the fact that default actions probably should work regardless of caller. 

Alright, done.